### PR TITLE
DDF-04288 Fix MenuItem overlap in the Intrigue location dropdown

### PR DESCRIPTION
**Pull request by jhunzik**
#### What does this PR do? 
Fixes the overlap that was found in the location dropdown in Intrigue. It also adds a tooltip to each MenuItem so the user can still read the full location without having to select it.
#### Who is reviewing it? 
@Kjames5269 
@emmberk 

#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.

@djblue 
@vinamartin

#### How should this be tested?
- Verify that long location names no longer overflow to the next line in Intrigue's location dropdown.
- Make sure that hovering over each suggestion displays a tooltip with the full location name.

#### What are the relevant tickets?

Fixes: #4288 

#### Screenshots
Before:
image
After:
image

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/menu/menu.js [hunk 1]
```diff
@@ -130,6 +130,7 @@ class MenuItem extends React.Component {
     } = this.props
     return (
       <div
+        title={value.name}
```
[the remark this answers is not in the document]
Comment (jhunzik) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/menu/menu.js]: We don't have the same UI element in the release branch. I'm not sure if we use this React component anywhere else, but I'll check.

[the remark this answers is not in the document]
Comment (vinamartin) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/menu/menu.js]: I would guess that it lives somewhere else on the release branch.

[the remark this answers is not in the document]
Comment (jhunzik) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/menu/menu.js]: Looks like it's not an issue on the release branch
screen shot 2019-02-01 at 10 40 46 am

**Comment by jhunzik** [comment]
build now

**Comment by vinamartin** [comment]
✅ Manually verified!

**Pull request merged**